### DISCUSSION TITLE
Cuándo aparece la visualización de cáracteres en una componente texto?

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#220] [2. Questions and Answers] [answered]
Hola
Me he percatado que aparece el contador de caracteres pero no sé cuál es la regla:

1. Aparece cuando el campo ha completado mas de la mitad del máximo?, en este caso se ve que son 61 de 120, como tambien podría ser 2501 de 5000).
2. Aparece sólo cuando han pasado mas de 60 caracteres

En caso que sea uno o lo, es factible que muestre el contador en cualquier momento o correspondería a una feature?.

Gracias

**Comment by acedward** [comment, marked as answer]
@NicolasPLG, la idea de este contador es que el usuario sepa que está cerca del límite. 

La razón que no aparece al comienzo es porque no queríamos saturar la interfaz con más información - siendo que este contador casí nunca debería aparecer - normalmente los límites deberían ser superiores al promedio de uso.

**Reply by NicolasPLG** [reply]
hola @acedward entiendo, entonces es casi al final. Es para tenerlo en cuenta, ya que probablemente algunos clientes futuros y/o presentas quieran que se muestre el contador de caracteres (en caso de dar una receta o solicitud). Etc.

Cerraré este tema y si aparece algo similar lo tomaré en referencia.
